Makefile

```diff
@@ -392,30 +402,48 @@ builddep: govendor
 genlocalfiles: .proto .bindata
 
 .PHONY: clean
-clean: skydive.cleanup test.functionals.cleanup dpdk.cleanup contribs.cleanup
-	grep path vendor/vendor.json | perl -pe 's|.*": "(.*?)".*|\1|g' | xargs -n 1 go clean -i >/dev/null 2>&1 || true
+clean: skydive.clean test.functionals.clean dpdk.clean contribs.clean govendor.clean
+
+.PHONY: fullclean
+fullclean: clean govendor.fullclean
```

Comment:
As this both fixes the below issue and removes stuff you don't want to delete during 'make clean' it does make sense to add a new target to remove evething added during 'make'

Reply:
let's have only one,simple make clean
make clean is used only by dev ... it will be confused to have more than one
git clean -i  is here to help you cleanup your repo

Reply:
ok - have removed 'fullclean' target